lib/pages/chat/chat.dart

```diff
@@ -1231,8 +1235,8 @@ class ChatController extends State<Chat>
   }
 
   void onInputBarSubmitted(_) async {
-    await send();
     await Future.delayed(const Duration(milliseconds: 100));
+    await send();
```

Comment:
why we need this change?

Reply:
no need. let me remove it

Reply:
need this fix, because of the double vietnamese message is sended